.github/workflows/release.yml

```
@@ -0,0 +1,136 @@
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          mkdir -p artifacts
+          gh release download ${{ github.ref_name }} -D artifacts
+
+      - name: Generate latest.json
+        id: generate-latest
+        run: |
+          VERSION="${{ github.ref_name }}"
+          RELEASE_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)